sw/device/tests/sim_dv/sysrst_ctrl_in_irq_test.c

```diff
@@ -76,11 +76,7 @@ void sysrst_ctrl_input_change_detect(
 
   // Configure for input change.
   dif_sysrst_ctrl_input_change_t input_change;
-  if (expected_key_intr_src >= kDifSysrstCtrlKeyIntrStatusInputPowerButtonL2H) {
-    input_change = (dif_sysrst_ctrl_input_change_t)expected_key_intr_src << 1;
-  } else {
-    input_change = (dif_sysrst_ctrl_input_change_t)expected_key_intr_src;
-  }
+  input_change = (dif_sysrst_ctrl_input_change_t)expected_key_intr_src;
   dif_sysrst_ctrl_input_change_config_t config = {
       .input_changes = input_change,
```

Comment:
Nit. I would just remove the line above and do as below.
```suggestion
      .input_changes = (dif_sysrst_ctrl_input_change_t)expected_key_intr_src,
```

Reply:
Thanks! Updated.